ipywidgets/static/widgets/less/widgets.less

```diff
@@ -178,37 +180,34 @@
     }
 }
 
+.widget-colorpicker {
+    /* Button */
+    width : @widget-width-short;
+}
+
 .widget-button {
     /* Button */
-    width         : @widget-width-short;
+    width : @widget-width-short;
 }
 
 .widget-text {
     /* Textbox */
-    width  : @widget-width;
-    margin : 0px;
+    width : @widget-width;
 }
 
 .widget-textarea {
     /* TextAreaView */
-    width  : @widget-width;
-    margin : 0px;
+    width : @widget-width;
 }
 
 .widget-listbox {
     /* Listbox */
-    width         : @widget-width;
-    margin-bottom : 0px;
-    /* Bootstrap does some weird stuff with the margins of select boxes.  The 
-    specifity of the media selector needs to be overwritten using important */
-    margin-left   : 0px !important; 
-    margin-right  : 0px !important;
+    width : @widget-width;
```

Comment:
I was thinking that `ipy-widget` makes the most sense as `ipy` is the org
and `widget` is the project. You could imagine `ipy-foo` and `ipy-bar`
eventually from seperate repos.

On Thu, Oct 29, 2015 at 5:29 PM, Sylvain Corlay <notifications@github.com>
wrote:

> In ipywidgets/static/widgets/less/widgets.less
> <https://github.com/ipython/ipywidgets/pull/216#discussion_r43446895>:
>
> >  }
> >
> >  .widget-listbox {
> >      /* Listbox */
> > -    width         : @widget-width;
> > -    margin-bottom : 0px;
> > -    /* Bootstrap does some weird stuff with the margins of select boxes.  The
> > -    specifity of the media selector needs to be overwritten using important */
> > -    margin-left   : 0px !important;
> > -    margin-right  : 0px !important;
> > +    width : @widget-width;
>
> Your change here was removed in the rebase. I can ad it back I guess but,
> it should probably be 2px.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/216/files#r43446895>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com